multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -1,114 +1,83 @@
 from typing import List, Optional
 
-from .constants import BINARY, DATA, ENVIRONMENT, MODEL, MULTICLASS, OPTIMIZATION, REGRESSION
+from .constants import (
+    BEST_QUALITY,
+    BINARY,
+    DATA,
+    DEFAULT,
+    ENVIRONMENT,
+    HIGH_QUALITY_FAST_INFERENCE,
+    MEDIUM_QUALITY_FASTER_INFERENCE,
+    MODEL,
+    MULTICLASS,
+    OPTIMIZATION,
+    REGRESSION,
+)
 from .registry import Registry
 
 automm_presets = Registry("automm_presets")
 matcher_presets = Registry("matcher_presets")
 
 
 @automm_presets.register()
-def high_quality_fast_inference():
-    return {
+def default(presets: str = DEFAULT, hpo: bool = False):
+    hyperparameters = {
         "model.names": [
             "categorical_mlp",
             "numerical_mlp",
             "timm_image",
             "hf_text",
             "fusion_mlp",
         ],
-        "model.hf_text.checkpoint_name": "google/electra-base-discriminator",
-        "model.timm_image.checkpoint_name": "swin_base_patch4_window7_224",
         "env.num_workers": 2,
     }
+    hyperparameter_tune_kwargs = None
+
+    if presets in [HIGH_QUALITY_FAST_INFERENCE, DEFAULT]:
+        if hpo:
+            pass
+        else:
+            hyperparameters.update(
+                {
+                    "model.hf_text.checkpoint_name": "google/electra-base-discriminator",
+                    "model.timm_image.checkpoint_name": "swin_base_patch4_window7_224",
+                }
+            )
+    elif presets == MEDIUM_QUALITY_FASTER_INFERENCE:
+        hyperparameters.update(
+            {
+                "model.hf_text.checkpoint_name": "google/electra-small-discriminator",
+                "model.timm_image.checkpoint_name": "swin_small_patch4_window7_224",
```

Comment:
For the quick build setting of image classification, we recommend to use `mobilenetv3_large_100 ` before, we might want to stay consistent.

Reply:
Changed to `mobilenetv3_large_100`.